tests/lib/tools/suite/fs-state/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: smoke test for the fs test tool
+execute: |
```

Comment:
I think it's somewhat repetitive but could you also shellcheck the script here? It would help to ensure, without resorting to some other location, that this is done.

Reply:
@you mean to do # shellcheck source=tests/lib/tools/fs-state ?

Reply:
No, I mean to run `shellcheck` on the `fs-state` file.

Reply:
This isn't done yet.

Reply:
The file appears to be shellcheck-clean now.